# HG changeset patch
# User Jacob Dawid <jacob.dawid@googlemail.com>
# Date 1338490436 -7200
#      Thu May 31 20:53:56 2012 +0200
# Branch gui
# Node ID f86884be20fcf75dfdecb74203e1abfcf040c011
# Parent  06abf71d9083b043340ce5045bdf810ccf814236
Renamed all source files of the gui to lowercase and .cc to be conform with the octave sources.

diff --git a/gui/src/backend/OctaveLink.cpp b/gui/src/backend/octavelink.cc
rename from gui/src/backend/OctaveLink.cpp
rename to gui/src/backend/octavelink.cc
--- a/gui/src/backend/OctaveLink.cpp
+++ b/gui/src/backend/octavelink.cc
@@ -10,17 +10,17 @@
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  * GNU General Public License for more details.
  *
  * You should have received a copy of the GNU General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
-#include "OctaveLink.h"
+#include "octavelink.h"
 #include "load-path.h"
 #include "oct-env.h"
 #include <QDir>
 #include <QApplication>
 
 int octave_readline_hook ()
 {
   OctaveLink::instance ()->triggerUpdateHistoryModel ();
diff --git a/gui/src/backend/OctaveLink.h b/gui/src/backend/octavelink.h
rename from gui/src/backend/OctaveLink.h
rename to gui/src/backend/octavelink.h
--- a/gui/src/backend/OctaveLink.h
+++ b/gui/src/backend/octavelink.h
@@ -68,19 +68,19 @@
 #include <QString>
 #include <QStringList>
 #include <QVector>
 #include <QSemaphore>
 #include <QObject>
 #include <QStringListModel>
 #include <QTimer>
 
-#include "WorkspaceModel.h"
-#include "OctaveMainThread.h"
-#include "SymbolInformation.h"
+#include "workspacemodel.h"
+#include "octavemainthread.h"
+#include "symbolinformation.h"
 
 /**
   * \class OctaveLink
   * Manages a link to an octave instance.
   */
 class OctaveLink:public QObject
 {
   Q_OBJECT
diff --git a/gui/src/backend/OctaveMainThread.cpp b/gui/src/backend/octavemainthread.cc
rename from gui/src/backend/OctaveMainThread.cpp
rename to gui/src/backend/octavemainthread.cc
--- a/gui/src/backend/OctaveMainThread.cpp
+++ b/gui/src/backend/octavemainthread.cc
@@ -10,18 +10,18 @@
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  * GNU General Public License for more details.
  *
  * You should have received a copy of the GNU General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
-#include "OctaveMainThread.h"
-#include "OctaveLink.h"
+#include "octavemainthread.h"
+#include "octavelink.h"
 
 OctaveMainThread::OctaveMainThread (QObject * parent):QThread (parent)
 {
 }
 
 void
 OctaveMainThread::run ()
 {
diff --git a/gui/src/backend/OctaveMainThread.h b/gui/src/backend/octavemainthread.h
rename from gui/src/backend/OctaveMainThread.h
rename to gui/src/backend/octavemainthread.h
diff --git a/gui/src/backend/SymbolInformation.h b/gui/src/backend/symbolinformation.h
rename from gui/src/backend/SymbolInformation.h
rename to gui/src/backend/symbolinformation.h
diff --git a/gui/src/editor/FileEditor.cpp b/gui/src/editor/fileeditor.cc
rename from gui/src/editor/FileEditor.cpp
rename to gui/src/editor/fileeditor.cc
--- a/gui/src/editor/FileEditor.cpp
+++ b/gui/src/editor/fileeditor.cc
@@ -10,17 +10,17 @@
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  * GNU General Public License for more details.
  *
  * You should have received a copy of the GNU General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
-#include "FileEditor.h"
+#include "fileeditor.h"
 #include <QVBoxLayout>
 #include <QApplication>
 #include <QFile>
 #include <QFont>
 #include <QFileDialog>
 #include <QMessageBox>
 #include <QStyle>
 #include <QTextStream>
diff --git a/gui/src/editor/FileEditor.h b/gui/src/editor/fileeditor.h
rename from gui/src/editor/FileEditor.h
rename to gui/src/editor/fileeditor.h
--- a/gui/src/editor/FileEditor.h
+++ b/gui/src/editor/fileeditor.h
@@ -13,19 +13,19 @@
  *
  * You should have received a copy of the GNU General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
 #ifndef FILEEDITORMDISUBWINDOW_H
 #define FILEEDITORMDISUBWINDOW_H
 
-#include "MainWindow.h"
-#include "FileEditorInterface.h"
-#include "FileEditorTab.h"
+#include "mainwindow.h"
+#include "fileeditorinterface.h"
+#include "fileeditortab.h"
 
 #include <QToolBar>
 #include <QAction>
 #include <QMenuBar>
 #include <QStatusBar>
 #include <QCloseEvent>
 #include <QTabWidget>
 #include <Qsci/qsciapis.h>
diff --git a/gui/src/editor/FileEditorInterface.h b/gui/src/editor/fileeditorinterface.h
rename from gui/src/editor/FileEditorInterface.h
rename to gui/src/editor/fileeditorinterface.h
diff --git a/gui/src/editor/FileEditorTab.cpp b/gui/src/editor/fileeditortab.cc
rename from gui/src/editor/FileEditorTab.cpp
rename to gui/src/editor/fileeditortab.cc
--- a/gui/src/editor/FileEditorTab.cpp
+++ b/gui/src/editor/fileeditortab.cc
@@ -10,18 +10,18 @@
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  * GNU General Public License for more details.
  *
  * You should have received a copy of the GNU General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
-#include "FileEditorTab.h"
-#include "FileEditor.h"
+#include "fileeditortab.h"
+#include "fileeditor.h"
 #include <QMessageBox>
 #include <QVBoxLayout>
 
 FileEditorTab::FileEditorTab(FileEditor *fileEditor)
   : QWidget ((QWidget*)fileEditor)
 {
   QSettings *settings = ResourceManager::instance ()->settings ();
   m_fileEditor = fileEditor;
diff --git a/gui/src/editor/FileEditorTab.h b/gui/src/editor/fileeditortab.h
rename from gui/src/editor/FileEditorTab.h
rename to gui/src/editor/fileeditortab.h
diff --git a/gui/src/editor/lexeroctavegui.cpp b/gui/src/editor/lexeroctavegui.cc
rename from gui/src/editor/lexeroctavegui.cpp
rename to gui/src/editor/lexeroctavegui.cc
diff --git a/gui/src/editor/lexeroctavegui.h b/gui/src/editor/lexeroctavegui.h
--- a/gui/src/editor/lexeroctavegui.h
+++ b/gui/src/editor/lexeroctavegui.h
@@ -13,17 +13,17 @@
  *
  * You should have received a copy of the GNU General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
 #ifndef LEXEROCTAVE_H
 #define LEXEROCTAVE_H
 
-#include "ResourceManager.h"
+#include "resourcemanager.h"
 #include <QObject>
 #include <Qsci/qsciglobal.h>
 #include <Qsci/qscilexer.h>
 
 
 class LexerOctaveGui : public QsciLexer
 {
     Q_OBJECT
diff --git a/gui/src/FilesDockWidget.cpp b/gui/src/filesdockwidget.cc
rename from gui/src/FilesDockWidget.cpp
rename to gui/src/filesdockwidget.cc
--- a/gui/src/FilesDockWidget.cpp
+++ b/gui/src/filesdockwidget.cc
@@ -10,18 +10,18 @@
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  * GNU General Public License for more details.
  *
  * You should have received a copy of the GNU General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
-#include "ResourceManager.h"
-#include "FilesDockWidget.h"
+#include "resourcemanager.h"
+#include "filesdockwidget.h"
 
 #include <QApplication>
 #include <QFileInfo>
 #include <QCompleter>
 #include <QSettings>
 #include <QProcess>
 #include <QDebug>
 
diff --git a/gui/src/FilesDockWidget.h b/gui/src/filesdockwidget.h
rename from gui/src/FilesDockWidget.h
rename to gui/src/filesdockwidget.h
diff --git a/gui/src/HistoryDockWidget.cpp b/gui/src/historydockwidget.cc
rename from gui/src/HistoryDockWidget.cpp
rename to gui/src/historydockwidget.cc
--- a/gui/src/HistoryDockWidget.cpp
+++ b/gui/src/historydockwidget.cc
@@ -10,17 +10,17 @@
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  * GNU General Public License for more details.
  *
  * You should have received a copy of the GNU General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
-#include "HistoryDockWidget.h"
+#include "historydockwidget.h"
 #include <QVBoxLayout>
 
 HistoryDockWidget::HistoryDockWidget (QWidget * parent):QDockWidget (parent)
 {
   setObjectName ("HistoryDockWidget");
   construct ();
 }
 
diff --git a/gui/src/HistoryDockWidget.h b/gui/src/historydockwidget.h
rename from gui/src/HistoryDockWidget.h
rename to gui/src/historydockwidget.h
--- a/gui/src/HistoryDockWidget.h
+++ b/gui/src/historydockwidget.h
@@ -17,17 +17,17 @@
 
 #ifndef HISTORYDOCKWIDGET_H
 #define HISTORYDOCKWIDGET_H
 
 #include <QDockWidget>
 #include <QLineEdit>
 #include <QListView>
 #include <QSortFilterProxyModel>
-#include "OctaveLink.h"
+#include "octavelink.h"
 
 class HistoryDockWidget:public QDockWidget
 {
 Q_OBJECT
 public:
   HistoryDockWidget (QWidget *parent = 0);
   void updateHistory (QStringList history);
 
diff --git a/gui/src/MainWindow.cpp b/gui/src/mainwindow.cc
rename from gui/src/MainWindow.cpp
rename to gui/src/mainwindow.cc
--- a/gui/src/MainWindow.cpp
+++ b/gui/src/mainwindow.cc
@@ -21,19 +21,19 @@
 #include <QSettings>
 #include <QStyle>
 #include <QToolBar>
 #include <QDesktopServices>
 #include <QFileDialog>
 #include <QMessageBox>
 #include <QIcon>
 
-#include "MainWindow.h"
-#include "FileEditor.h"
-#include "SettingsDialog.h"
+#include "mainwindow.h"
+#include "fileeditor.h"
+#include "settingsdialog.h"
 
 MainWindow::MainWindow (QWidget * parent):QMainWindow (parent)
 {
   // We have to set up all our windows, before we finally launch octave.
   construct ();
   OctaveLink::instance ()->launchOctave();
 }
 
diff --git a/gui/src/MainWindow.h b/gui/src/mainwindow.h
rename from gui/src/MainWindow.h
rename to gui/src/mainwindow.h
--- a/gui/src/MainWindow.h
+++ b/gui/src/mainwindow.h
@@ -27,28 +27,28 @@
 #include <QToolBar>
 #include <QQueue>
 #include <QMdiSubWindow>
 #include <QCloseEvent>
 #include <QToolButton>
 #include <QComboBox>
 
 // Editor includes
-#include "FileEditorInterface.h"
+#include "fileeditorinterface.h"
 
 // QTerminal includes
 #include "QTerminal.h"
 
 // Own includes
-#include "ResourceManager.h"
-#include "OctaveLink.h"
-#include "WorkspaceView.h"
-#include "HistoryDockWidget.h"
-#include "FilesDockWidget.h"
-#include "TerminalDockWidget.h"
+#include "resourcemanager.h"
+#include "octavelink.h"
+#include "workspaceview.h"
+#include "historydockwidget.h"
+#include "filesdockwidget.h"
+#include "terminaldockwidget.h"
 
 /**
   * \class MainWindow
   *
   * Represents the main window.
   */
 class MainWindow:public QMainWindow
 {
diff --git a/gui/src/OctaveGUI.cpp b/gui/src/octavegui.cc
rename from gui/src/OctaveGUI.cpp
rename to gui/src/octavegui.cc
--- a/gui/src/OctaveGUI.cpp
+++ b/gui/src/octavegui.cc
@@ -13,19 +13,19 @@
  *
  * You should have received a copy of the GNU General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
 #include <QtGui/QApplication>
 #include <QTranslator>
 #include <QSettings>
-#include "WelcomeWizard.h"
-#include "ResourceManager.h"
-#include "MainWindow.h"
+#include "welcomewizard.h"
+#include "resourcemanager.h"
+#include "mainwindow.h"
 
 int
 main (int argc, char *argv[])
 {
   QApplication application (argc, argv);
   while (true)
     {
       if (ResourceManager::instance ()->isFirstRun ())
diff --git a/gui/src/ResourceManager.cpp b/gui/src/resourcemanager.cc
rename from gui/src/ResourceManager.cpp
rename to gui/src/resourcemanager.cc
--- a/gui/src/ResourceManager.cpp
+++ b/gui/src/resourcemanager.cc
@@ -10,17 +10,17 @@
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  * GNU General Public License for more details.
  *
  * You should have received a copy of the GNU General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
-#include "ResourceManager.h"
+#include "resourcemanager.h"
 #include <QFile>
 #include <QNetworkProxy>
 
 ResourceManager ResourceManager::m_singleton;
 
 ResourceManager::ResourceManager ()
 {
   m_settings = 0;
diff --git a/gui/src/ResourceManager.h b/gui/src/resourcemanager.h
rename from gui/src/ResourceManager.h
rename to gui/src/resourcemanager.h
diff --git a/gui/src/SettingsDialog.cpp b/gui/src/settingsdialog.cc
rename from gui/src/SettingsDialog.cpp
rename to gui/src/settingsdialog.cc
--- a/gui/src/SettingsDialog.cpp
+++ b/gui/src/settingsdialog.cc
@@ -10,34 +10,34 @@
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  * GNU General Public License for more details.
  *
  * You should have received a copy of the GNU General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
-#include "ResourceManager.h"
-#include "SettingsDialog.h"
-#include "ui_SettingsDialog.h"
+#include "resourcemanager.h"
+#include "settingsdialog.h"
+#include "ui_settingsdialog.h"
 #include <QSettings>
 
 SettingsDialog::SettingsDialog (QWidget * parent):
 QDialog (parent), ui (new Ui::SettingsDialog)
 {
   ui->setupUi (this);
 
   QSettings *settings = ResourceManager::instance ()->settings ();
   ui->useCustomFileEditor->setChecked (settings->value ("useCustomFileEditor").toBool ());
   ui->customFileEditor->setText (settings->value ("customFileEditor").toString ());
   ui->editor_showLineNumbers->setChecked (settings->value ("editor/showLineNumbers",true).toBool () );
   ui->editor_highlightCurrentLine->setChecked (settings->value ("editor/highlightCurrentLine",true).toBool () );
   ui->editor_codeCompletion->setChecked (settings->value ("editor/codeCompletion",true).toBool () );
   ui->editor_fontName->setCurrentFont (QFont (settings->value ("editor/fontName","Courier").toString()) );
-  ui->editor_fontSize->setValue (settings->value ("editor/fontSize",10).toInt ());  
+  ui->editor_fontSize->setValue (settings->value ("editor/fontSize",10).toInt ());
   ui->editor_longWindowTitle->setChecked (settings->value ("editor/longWindowTitle",true).toBool ());
   ui->terminal_fontName->setCurrentFont (QFont (settings->value ("terminal/fontName","Courier").toString()) );
   ui->terminal_fontSize->setValue (settings->value ("terminal/fontSize",10).toInt ());
   ui->showFilenames->setChecked (settings->value ("showFilenames").toBool());
   ui->showFileSize->setChecked (settings->value ("showFileSize").toBool());
   ui->showFileType->setChecked (settings->value ("showFileType").toBool());
   ui->showLastModified->setChecked (settings->value ("showLastModified").toBool());
   ui->showHiddenFiles->setChecked (settings->value ("showHiddenFiles").toBool());
diff --git a/gui/src/SettingsDialog.h b/gui/src/settingsdialog.h
rename from gui/src/SettingsDialog.h
rename to gui/src/settingsdialog.h
diff --git a/gui/src/SettingsDialog.ui b/gui/src/settingsdialog.ui
rename from gui/src/SettingsDialog.ui
rename to gui/src/settingsdialog.ui
diff --git a/gui/src/src.pro b/gui/src/src.pro
--- a/gui/src/src.pro
+++ b/gui/src/src.pro
@@ -69,45 +69,45 @@ win32-msvc* {
     QMAKE_CXXFLAGS += -wd4244
 }
 
 QMAKE_LFLAGS        += $$LFLAGS $$system(mkoctfile -p RLD_FLAG)
 QMAKE_CXXFLAGS      += $$INCFLAGS
 
 # Files associated with the project:
 SOURCES +=\
-    editor/lexeroctavegui.cpp \
-    MainWindow.cpp \
-    WorkspaceView.cpp \
-    HistoryDockWidget.cpp \
-    FilesDockWidget.cpp \
-    SettingsDialog.cpp \
-    OctaveGUI.cpp \
-    ResourceManager.cpp \
-    backend/OctaveLink.cpp \
-    backend/OctaveMainThread.cpp \
-    WelcomeWizard.cpp \
-    editor/FileEditor.cpp \
-    WorkspaceModel.cpp \
-    editor/FileEditorTab.cpp \
-    TerminalDockWidget.cpp
+    backend/octavelink.cc \
+    backend/octavemainthread.cc \
+    editor/lexeroctavegui.cc \
+    editor/fileeditor.cc \
+    editor/fileeditortab.cc \
+    mainwindow.cc \
+    workspaceview.cc \
+    historydockwidget.cc \
+    filesdockwidget.cc \
+    settingsdialog.cc \
+    octavegui.cc \
+    resourcemanager.cc \
+    welcomewizard.cc \
+    workspacemodel.cc \
+    terminaldockwidget.cc
 
 HEADERS += \
+    backend/octavelink.h \
+    backend/octavemainthread.h \
+    backend/symbolinformation.h \
     editor/lexeroctavegui.h \
-    MainWindow.h \
-    WorkspaceView.h \
-    HistoryDockWidget.h \
-    FilesDockWidget.h \
-    SettingsDialog.h \
-    ResourceManager.h \
-    backend/OctaveLink.h \
-    backend/OctaveMainThread.h \
-    WelcomeWizard.h \
-    editor/FileEditor.h \
-    WorkspaceModel.h \
-    editor/FileEditorInterface.h \
-    editor/FileEditorTab.h \
-    TerminalDockWidget.h \
-    backend/SymbolInformation.h
+    editor/fileeditor.h \
+    editor/fileeditorinterface.h \
+    editor/fileeditortab.h \
+    mainwindow.h \
+    workspaceview.h \
+    historydockwidget.h \
+    filesdockwidget.h \
+    settingsdialog.h \
+    resourcemanager.h \
+    welcomewizard.h \
+    workspacemodel.h \
+    terminaldockwidget.h
 
 FORMS += \
-    SettingsDialog.ui \
-    WelcomeWizard.ui
+    settingsdialog.ui \
+    welcomewizard.ui
diff --git a/gui/src/TerminalDockWidget.cpp b/gui/src/terminaldockwidget.cc
rename from gui/src/TerminalDockWidget.cpp
rename to gui/src/terminaldockwidget.cc
--- a/gui/src/TerminalDockWidget.cpp
+++ b/gui/src/terminaldockwidget.cc
@@ -10,17 +10,17 @@
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  * GNU General Public License for more details.
  *
  * You should have received a copy of the GNU General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
-#include "TerminalDockWidget.h"
+#include "terminaldockwidget.h"
 
 TerminalDockWidget::TerminalDockWidget (QTerminal *terminal, QWidget *parent)
   : QDockWidget (parent)
 {
   setObjectName ("TerminalDockWidget");
   setWindowTitle (tr ("Command Window"));
   setWidget (terminal);
 
diff --git a/gui/src/TerminalDockWidget.h b/gui/src/terminaldockwidget.h
rename from gui/src/TerminalDockWidget.h
rename to gui/src/terminaldockwidget.h
diff --git a/gui/src/WelcomeWizard.cpp b/gui/src/welcomewizard.cc
rename from gui/src/WelcomeWizard.cpp
rename to gui/src/welcomewizard.cc
--- a/gui/src/WelcomeWizard.cpp
+++ b/gui/src/welcomewizard.cc
@@ -10,18 +10,18 @@
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  * GNU General Public License for more details.
  *
  * You should have received a copy of the GNU General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
-#include "WelcomeWizard.h"
-#include "ui_WelcomeWizard.h"
+#include "welcomewizard.h"
+#include "ui_welcomewizard.h"
 
 WelcomeWizard::WelcomeWizard (QWidget *parent) :
   QDialog (parent),
   ui (new Ui::WelcomeWizard)
 {
   ui->setupUi (this);
   connect (ui->nextButton1, SIGNAL (clicked ()), this, SLOT (next ()));
   connect (ui->nextButton2, SIGNAL (clicked ()), this, SLOT (next ()));
diff --git a/gui/src/WelcomeWizard.h b/gui/src/welcomewizard.h
rename from gui/src/WelcomeWizard.h
rename to gui/src/welcomewizard.h
diff --git a/gui/src/WelcomeWizard.ui b/gui/src/welcomewizard.ui
rename from gui/src/WelcomeWizard.ui
rename to gui/src/welcomewizard.ui
diff --git a/gui/src/WorkspaceModel.cpp b/gui/src/workspacemodel.cc
rename from gui/src/WorkspaceModel.cpp
rename to gui/src/workspacemodel.cc
--- a/gui/src/WorkspaceModel.cpp
+++ b/gui/src/workspacemodel.cc
@@ -10,20 +10,20 @@
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  * GNU General Public License for more details.
  *
  * You should have received a copy of the GNU General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
-#include "WorkspaceModel.h"
+#include "workspacemodel.h"
 #include <QTreeWidget>
 #include <QTime>
-#include "OctaveLink.h"
+#include "octavelink.h"
 
 WorkspaceModel::WorkspaceModel(QObject *parent)
   : QAbstractItemModel(parent)
 {
   QList<QVariant> rootData;
   rootData << tr ("Name") << tr ("Type") << tr ("Value");
   _rootItem = new TreeItem(rootData);
 }
diff --git a/gui/src/WorkspaceModel.h b/gui/src/workspacemodel.h
rename from gui/src/WorkspaceModel.h
rename to gui/src/workspacemodel.h
diff --git a/gui/src/WorkspaceView.cpp b/gui/src/workspaceview.cc
rename from gui/src/WorkspaceView.cpp
rename to gui/src/workspaceview.cc
--- a/gui/src/WorkspaceView.cpp
+++ b/gui/src/workspaceview.cc
@@ -10,17 +10,17 @@
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  * GNU General Public License for more details.
  *
  * You should have received a copy of the GNU General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
-#include "WorkspaceView.h"
+#include "workspaceview.h"
 #include <QHBoxLayout>
 #include <QVBoxLayout>
 #include <QPushButton>
 
 WorkspaceView::WorkspaceView (QWidget * parent) : QDockWidget
   (parent)
 {
   setObjectName ("WorkspaceView");
diff --git a/gui/src/WorkspaceView.h b/gui/src/workspaceview.h
rename from gui/src/WorkspaceView.h
rename to gui/src/workspaceview.h
--- a/gui/src/WorkspaceView.h
+++ b/gui/src/workspaceview.h
@@ -16,17 +16,17 @@
  */
 
 #ifndef WORKSPACEVIEW_H
 #define WORKSPACEVIEW_H
 
 #include <QDockWidget>
 #include <QTreeView>
 #include <QSemaphore>
-#include "OctaveLink.h"
+#include "octavelink.h"
 
 class WorkspaceView:public QDockWidget
 {
   Q_OBJECT
 public:
   WorkspaceView (QWidget * parent = 0);
 
 public slots:
